core/src/main/scala/kafka/server/ControllerMutationQuotaManager.scala

```diff
@@ -63,7 +63,7 @@ abstract class AbstractControllerMutationQuota(private val time: Time) extends C
   protected var lastRecordedTimeMs = 0L
```

Comment:
It looks like there is no other class uses this variable directly. Can we update it as private? Thanks.

Reply:
+1